### ISSUE TITLE
knitr2html no response, knitr2pdf required system manager

### ISSUE THREAD

**Issue by englianhu** [yihui/knitr#1107]
How to `knitr2pdf` on Linux? I installed pandocs.
![untitled1](https://cloud.githubusercontent.com/assets/7227582/10159526/bb25c1d2-66d3-11e5-8763-afb3c9567b5e.jpg)

```{r}
> Sys.info()
                              sysname                               release                               version                              nodename 
                              "Linux"           "3.10.0-123.8.1.el7.x86_64" "#1 SMP Mon Sep 22 19:06:58 UTC 2014"                           "scibrokes" 
                              machine                                 login                                  user                        effective_user 
                             "x86_64"                             "unknown"                              "ryoeng"                              "ryoeng"
```

Secondly, `knitr2html` doesn't work since yesterday. Any solution?
![untitled2](https://cloud.githubusercontent.com/assets/7227582/10159628/5c373614-66d4-11e5-86a6-6cd6288c47d6.jpg)

**Comment by yihui** [comment]
Please read the help page `?knit2pdf`. It was not designed to work with R Markdown, but with Rnw instead.

For `knit2html`, I don't know what you mean by "doesn't work" (that is the most vague description of a problem). I'm not sure if you have read the help page `?knit2html`, and it is not designed to work with R Markdown v2 as the help page has clarified.

**Comment by englianhu** [comment]
The icon changed to knit button always non-workable similar with last month (unable to view the output in panel as well). The I tried to upgrade to V0.99.484 but keep same issue. Its work on RConsole but doesn't work on **knit** here I upload the error page.

![knit](https://cloud.githubusercontent.com/assets/7227582/10183821/35ae8542-676f-11e5-9be2-9fdf0fd925aa.jpg)

AThere is a blank page when I tried to post to RPub just now.
http://rpubs.com/englianhu/testing
